applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -151,17 +151,17 @@ static void stream_started_cb(struct bt_audio_stream *stream)
 	LOG_INF("Stream index %d started", active_stream_index);
 }
 
-static void stream_stopped_cb(struct bt_audio_stream *stream)
+static void stream_stopped_cb(struct bt_bap_stream *stream, uint8_t reason)
 {
 	int ret;
 
 	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
 	ERR_CHK(ret);
 
-	LOG_INF("Stream index %d stopped", active_stream_index);
+	LOG_INF("Stream index %d stopped. Reason: %d", active_stream_index, reason);
```

Comment:
I think the reason should be DBG

Reply:
As discussed on Teams, this will be addressed when we do the architecture rewrite